<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_process</class>
 <widget class="QDialog" name="data_process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QTableWidget" name="date_data">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>441</y>
     <width>221</width>
     <height>261</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>321</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Time</string>
    </property>
   </column>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ZapfChancery-Medium</family>
     <pointsize>36</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Data processing performance</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="generate_data">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Month</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Performance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Target</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Achievement rate</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="data">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>400</y>
     <width>561</width>
     <height>301</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Machine</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Performance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Target</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Achievement rate</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Averger</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>30</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Machine 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Machine 2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose machine:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Month:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="choose_month">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>February</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>120</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
